tests/unit/data/test_extract_metadata.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+from iris.coords import DimCoord
+from iris.cube import Cube, CubeList
+import numpy as np
+from clean_air import data
+from edr_server.core.models.metadata import CollectionMetadata
+
+class TestExtractMetadata:
+	@staticmethod
+	@pytest.fixture
+	def cube_1():
+		latitude = DimCoord(np.linspace(-90, 90, 4),
+                    standard_name='latitude',
+                    units='degrees')
+		longitude = DimCoord(np.linspace(45, 360, 8),
+					standard_name='longitude',
+					units='degrees')
+		time = DimCoord(np.linspace(1, 24, 24),
+					standard_name='time',
+					units="hours since 1970-01-01 00:00:00")
+		height = DimCoord(3.5, 
+					standard_name="height",
+					units="m",
+					attributes={'positive': 'up'})
+		cube = Cube(np.zeros((4, 8, 24), np.float32),
+					standard_name="mass_concentration_of_ozone_in_air",
+					units="ug/m3",
+					dim_coords_and_dims=[(latitude, 0),
+										(longitude, 1),
+										(time, 2)])
```

Comment:
One minor thing, the formatting in this file is a bit awkward. In Pycharm I'd just use the autoformatter, apparently VSCode has some options: https://code.visualstudio.com/docs/python/editing#_formatting

Has this file been run through a formatter?

Reply:
Won't let me install any :( 
![image](https://user-images.githubusercontent.com/83035175/188884327-8f60a092-cee4-4750-a22e-9b2fa77d3f6e.png)
Autoformatter works fine for javascript. Remind me is there a different location/method to install it?

Reply:
Looks like it's trying to install into the system-wide location, which you won't have permissions to do. Try using `/bin/python3 -m pip install --user -U autopep8` (The `--user` flag will install it just for you in your user area)